src/test/setup.js

```diff
@@ -29,6 +42,9 @@ afterEach(() => {
   jest.restoreAllMocks();
   jest.clearAllTimers();
   jest.useRealTimers();
+
+  // Do the same with fetch mocks
+  fetchMock.mockReset();
```

Comment:
I'm not sure this is strictly needed because we add a new version of the sandbox for each test. But better be safe than sorry :)

Also I'm not deleting the `fetch` property from the global. This seems to work fine for now but we might need to adjust in the future.